README.md

```diff
@@ -1,5 +1,10 @@
 # Perseo-core (EPL server)
 
+[![FIWARE Processing](https://nexus.lab.fiware.org/static/badges/chapters/processing.svg)](https://www.fiware.org/developers/catalogue/)
```

Comment:
Is https://nexus.lab.fiware.org/static/badges/chapters/processing.svg working?

I'm getting a 404 error...